lib/js/components/Drawer/Drawer.vue

```diff
@@ -8,15 +8,47 @@
 
 <style lang="scss" scoped>
 @import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/media-queries';
 
 .drawer {
-	border-left: 1px solid $color-neutral-border-weak;
 	height: 100%;
+
+	@media #{breakpoint-s()} {
+		&.-positionLeft {
+			border-right: 1px solid $color-neutral-border-weak;
```

Comment:
instead of defining all `border` properties in both modifier blocks, you could style on `border-right-width` and define common styles in upper block